### DISCUSSION TITLE
Creacion repositorio TPI3

### DISCUSSION THREAD

**Discussion by lrocampo** [dds-jv/foro#441] [Consultas generales] [answered]
Buenas! Ingresé al link para que se genere mi repositorio del TPI3 y me salto este problema
<img width="1440" alt="image" src="https://github.com/dds-jv/foro/assets/88249238/da077c5d-7b71-47fb-9199-fbdff5512584">
Y no se qué hacer :P

**Comment by flbulgarelli** [comment]
¡Hola! Estuve investigando el tema y no tengo aún una solución, este error es nuevo. ¿Te llega a crear el repositorio?

**Reply by flbulgarelli** [reply, marked as answer]
:up: Parece que el problema es una actualización reciente en el funcionamiento de Github Classroom y la única forma que encontramos de permitir crear nuevas asignaciones fue crear una nueva versión del TPI. 

Acabamos de actualizar en el WebCampus el enlace al TPI; debería llevarte a `Calendarios - v2`. Probá resolver el ejercicio desde ese nuevo enlace. 

¡Saludos!
